crates/client/src/admin_websocket.rs

```diff
@@ -524,8 +524,11 @@ impl AdminWebsocket {
         }
     }
 
-    pub async fn agent_info(&self, cell_id: Option<CellId>) -> ConductorApiResult<Vec<String>> {
-        let msg = AdminRequest::AgentInfo { cell_id };
+    pub async fn agent_info(
```

Comment:
Should this name be plural throughout? `agent_info` -> `list_agent_infos` or just `agent_infos`?

Reply:
It looks like it was already inconsistent so maybe that's best left as a cleanup for a later PR, because I'd like to be able to backport this one.